Dockerfile

```diff
@@ -60,21 +15,12 @@ RUN mv ./argo-linux-amd64 /usr/local/bin/argo
 RUN argo version
 
 ####################################################################################################
-# sensor
-####################################################################################################
-FROM scratch as sensor
-COPY --from=base /usr/share/zoneinfo /usr/share/zoneinfo
-COPY --from=base /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/ca-certificates.crt
-COPY --from=sensor-base /usr/local/bin/argo /usr/local/bin/argo
-COPY dist/sensor /bin/sensor
-ENTRYPOINT [ "/bin/sensor" ]
-
-####################################################################################################
-# events-webhook
+# argo-events
 ####################################################################################################
-FROM scratch as events-webhook
+FROM scratch as argo-events
```

Comment:
I think distroless has TZ data built in

Reply:
I know. I just don't think there's no internal distroless support at this time. This can be easily changed later.